matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -532,6 +546,49 @@ internal class EventRelationsAggregationProcessor @Inject constructor(
         }
     }
 
+    private fun handleLiveLocation(realm: Realm,
+                                   event: Event,
+                                   content: MessageLiveLocationContent,
+                                   roomId: String,
+                                   isLocalEcho: Boolean) {
+        val locationSenderId = event.senderId ?: return
+
+        // We shouldn't process local echos
+        if (isLocalEcho) {
+            return
+        }
+
+        // A beacon info state event has to be sent before sending location
+        val beaconInfoEntity = CurrentStateEventEntity.getOrNull(realm, roomId, locationSenderId, EventType.STATE_ROOM_BEACON_INFO.first())
```

Comment:
Would it be possible to check for both the stable and unstable prefixes for compaibility reason? And using `first()`, we are not sure to use the correct prefix since the list may change over time.

Reply:
Oh, nice catch! Done.